R/dsl-generate.R

```diff
@@ -146,6 +165,13 @@ dsl_generate_domain <- function(dat, meta) {
       }
     }
   }
+
+  ## Same logic as model_combine_domain
```

Comment:
We might look to do the combination earlier perhaps? Not very straightforward to do so, though

Reply:
no real advantage of doing this actually